src/components/AssetsComponent.jsx

```diff
@@ -38,6 +40,7 @@ export class AssetsComponent extends Component {
   };
 
   handlePaginationChange = (e, { activePage }) => {
+    localStorage.setItem('activePage', activePage);

```

Comment:
Why not set the `activePage` in the store instead of localStorage? Something like:
```
export const setActivePage = activePage => ({
  type: SET_ACTIVE_PAGE,
  activePage
})
```
Then the activePage data can be retrieved from the store in whichever component that needs it. `localStorage` isn't the best solution here.

Also note that once you set the `activePage` in the store, the `activePage` state in this component ceases to exist - meaning that it should be deleted